src/modules/menu-category/menu-category.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseEntity, Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from "typeorm";
+
+@Entity('menu_categories')
+export class MenuCategory extends BaseEntity {
+  @PrimaryGeneratedColumn('increment')
+  id: number;
+  @Column({ type: 'varchar', name: 'name', length: 50 })
+  name: string;
+  @Column({ name: 'author_id', nullable: true })
```

Comment:
Here, you must put the column as nullable: false

Reply:
OK, i get it 🤖